nexus-repository-conan-it/pom.xml

```diff
@@ -86,6 +86,13 @@
       <version>${nxrm-version}</version>
       <type>zip</type>
       <scope>test</scope>
+      <exclusions>
+        <!-- Avoid leaking older r plugins from distro -->
```

Comment:
should be conan instead of r

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/124/commits/46b8144bda9cc54bda9adfa8a079098e3a7736c9